socorro/unittest/external/http/test_correlations.py

```diff
@@ -25,9 +27,20 @@
 
 
 class Response(object):
-    def __init__(self, content=None, status_code=200):
-        self.content = content
+    def __init__(self, content=None, status_code=200, gzip=False):
```

Comment:
fixed. I now call it `compress`. Why didn't I think of that in the first place?!